build.gradle

```diff
@@ -87,9 +77,15 @@ allprojects {
     apply from: "$rootDir/gradle/rat.gradle"
 
     repositories {
+        mavenLocal()
         jcenter()
         mavenCentral()
-        mavenLocal()
```

Comment:
This ensures the local version gets predence over the remote repos. Is this intentional ?